static/js/chart/draw_d3_map.ts

```diff
@@ -342,9 +342,7 @@ export function getProjection(
       projection = d3.geoEquirectangular().rotate([-100, 0]).precision(0.1);
       break;
     default:
-      projection = isUSAPlace
-        ? geo.geoAlbersUsaTerritories()
-        : d3.geoEquirectangular();
+      projection = isUSAPlace ? d3.geoAlbersUsa() : d3.geoEquirectangular();
```

Comment:
@chejennifer which page should I verify for this projection change?

Reply:
you can check any place that has US map charts - probably want to check US and a US state and US territory so maybe: 

/place/country/USA
/place/geoId/06
/place/geoId/72

Reply:
Reverted as the lib does not block the version update